Operational-Procedures/Scheduler/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _scheduler-operational-procedures:
+
+################################
+Scheduler Operational Procedures
+################################
+
+The Scheduler CSC is in charge of computing an observing queue for the observatory.
+There are two instances of the CSC running at all times, one dedicated to the Main Telescope and another to the Auxiliary Telescope, with indexes 1 and 2 respectively.
+
+Both CSCs share the exact same code-base, which means operating and throubleshooting shares several commonalities.
+
+This page focus on the operational aspects of the Scheduler CSC, in general.
+Any difference between the operation of MT and AT Scheduler will be appropriately highlighted. 
+
+For more details about the CSC, see the `Scheduler CSC documentation`_.
+
+.. _Scheduler CSC documentation: https://ts-scheduler.lsst.io
+
+If you are interested in the official scheduling algorithm, that is used to compute the observing queue, see the `Feature Based Scheduler documentation`_.
+
+.. _Feature Based Scheduler documentation: https://rubin-sim.lsst.io
+
+
+.. _scheduler-operational-procedures-overview:
+
+Overview
+========
+
+Before diving in the operational procedures, let's have a quick overview of how the Scheduler CSC operates.
+
+The Scheduler CSC can be divided into two main domains; *observatory interaction* and the *scheduling algorithm*.
+
+The observatory interaction part of the CSC is responsible for the business logic of the component.
+Its main tasks are:
+
+* configuring the component with a user-provided set of parameters,
+* interacting with the observatory control to update the observatory state,
+* gathering telemetry from the EFD, 
+* executing the scheduling algorithm to obtain a list of targets,
+* sending the targets to the ScriptQueue.
+
+The CSC defines an API for *scheduling algorithm*, providing a highly flexible architecture allowing for a range of applications to be designed and used.
+
+The official *scheduling algorithm* adopted by the project is the so-called "**Feature Based Scheduler**".
+The most relevant aspects of this application are:
+
+* scheduling routines are expressed in terms of *Features*, which are basically some kind of "information".
+* *Basis functions* computes rewards from a collection of *Features*.
+* *Surveys* linearly combine a collection of *Basis functions* to compute a final reward.
+* A collection of *Surveys* can be combined and, at any time, the survey with the highest reward value is selected to produce a target.
+
+This is illustred in :ref:`fig-fbs-architecture`.
+It is important for users to understand this hierarchical structure for throubleshooting the scheduler operation later.
+
+.. figure:: ./_static/fbs-architecture.png
```

Comment:
I think this just needs more information in general... or maybe a link to another document?
It might be nice to add more text in each box. For example, in the survey box, you could put "deep drilling" etc.
BasisFuction could include "metric calculator" or something, and then each feature could be "wind" "slew time" etc.

Another nice thing might be to add a dotted line to another box on each level to basically show there are infinite amounts of surveys, basis functions etc.

Reply:
I added a link to the FBS documentation page, I didn't want to spend too much time explaining it here.